src/Nethermind/Nethermind.Network.Discovery/NettyDiscoveryHandler.cs

```diff
@@ -92,6 +92,11 @@ public async void SendMsg(DiscoveryMsg discoveryMsg)
             return;
         }
 
+        if (msgBytes.Length > 1280)
+        {
+            _logger.Warn($"Attempting to send message larger than 1280 bytes. This is out of spec and may not work for all client. Msg: ${discoveryMsg}");
```

Comment:
please add if(_logger.Warn) guard. Convention to not create those strings if not needed.